paper-input-container.html

```diff
@@ -515,8 +515,7 @@
     _handleValue: function(inputElement) {
       var value = this._inputElementValue;
 
-      // type="number" hack needed because this.value is empty until it's valid
```

Comment:
But without my changes on master http://jsbin.com/kehimanape/1/edit?html,output its already broken I think.

I think the other jsbin pulls in a different version of something.